src/style.css

```
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.stars__row {
+  display: flex;
+}
+
+.star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.star :last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(-n+1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
```

you can combine these selectors
```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url(./images/star-active.svg);
}
```